README.md

```diff
@@ -1,5 +1,37 @@
 # md
 
+![summary.png](docs/images/summary.png)
```

Comment:
Should the image be stored as a separate file?
You can insert image into README.MD (ctr+v)